src/adapter/deconz/adapter/deconzAdapter.ts

```diff
@@ -393,14 +402,15 @@ class DeconzAdapter extends Adapter {
             req.txOptions = 0;
             req.radius = PARAM.PARAM.txRadius.DEFAULT_RADIUS;
 
-            this.driver
-                .enqueueSendDataRequest(req)
-                .then((result) => {})
-                .catch((error) => {});
+            try {
```

Comment:
@Nerivec I think this might break the `waitForData` below, since it now wait till this is completed.